### DISCUSSION TITLE
Tina Media has not yet been configured on this branch.

### DISCUSSION THREAD

**Discussion by jazbogross** [tinacms/tinacms#4295] [General]
I'm trying to set up repo-based media.

I've have the following in my config

  media: {
    tina: {
      publicFolder: "public",
      mediaRoot: "uploads",
    },
  },

I am working on the main branch.

But I am still getting the message:
Media needs to be turned on for this project.
[Sync Your Media In Tina Cloud.](https://app.tina.io/projects/c77846e7-fd5d-4c47-b5bd-f0cb63f84506/media)

and then:
Tina Media Not Configured
Tina Media has not yet been configured on this branch.
If you're looking to setup Tina Media, visit [our docs.](https://tina.io/docs/reference/media/overview/)

The docs tells me to:
To workaround this, you may need to enable media after merging Tina to your default branch.

But I'm not sure what that means. Can anyone help troubleshoot this? What information would I need to provide if any?

**Comment by stephen-universe** [comment]
I’m facing this exact issue!
Did you ever figure out a fix to this?

**Comment by phantomhell** [comment]
make your config like this:
`import { defineConfig } from "tinacms";

// Your tina will search for a branch with fallback to "main
const branch =
  process.env.PUBLIC_TINA_BRANCH ||
  process.env.CF_PAGES_BRANCH ||
  process.env.HEAD ||
  "main";

export default defineConfig({
  clientId: process.env.NEXT_PUBLIC_TINA_CLIENT_ID!, // required
  token: process.env.TINA_TOKEN!,                    // required
  branch,                                            // use const branch

  media: {
    tina: {
      publicFolder: "public",
      mediaRoot: "uploads",
      static: false,
    },
  },

  search: {
    tina: {
      indexerToken: process.env.TINA_SEARCH_TOKEN!,
      stopwordLanguages: ["eng"],
    },
    indexBatchSize: 100,
    maxSearchIndexFieldLength: 100,
  },

  build: {
    outputFolder: "admin",
    publicFolder: "public",
  },

  schema: {
    collections: [
   // you collections goes here
        ],
      },
    ],
  },
});`

**Reply by stephen-universe** [reply]
Yeah I was able to solve this problem.

The issue was that my schema file was set up as you stated which was confusing because media was still not being recognized but I later found the issue was due to the TinaCmS template. Somewhere in the template files, the media was set & turned off. I didn't go into searching how to turn it back on I simply just created a new barebones TinaCMS... copy & pasted my original schema and problem solved.

**Reply by ScottA38** [reply]
@stephen-universe Experiencing the same issue. When you say ‘created a new barebones TinaCMS’, do you mean that you ran `npx run create-tina-app` in your website repo, or that you created a new TinaCloud project?
